tide_site.module

```diff
@@ -588,6 +588,40 @@ function tide_site_form_alter(array &$form, FormStateInterface $form_state, $for
       }
     }
   }
+  // Add site prefix to redirect URL.
+  if ('redirect_redirect_form' === $form_id) {
+    $sites_term = 'sites';
+    // Get all primary sites ony, do not load sub-sites.
+    $term_tree = \Drupal::entityTypeManager()->getStorage('taxonomy_term')->loadTree(
+      $sites_term,
+      0,
+      1,
+      TRUE
+    );
+    // Build options list.
+    $options = [];
+
+    foreach ($term_tree as $term) {
+      if ($term instanceof Term) {
+        $options['site-' . $term->id()] = $term->getName();
```

Comment:
Just use the Term Id as an integer for the options id.

```php
$options[$term->id()] = $term->getName();
```

Will allow us to use the integer in validation and possibly AJAX calls to load the term by its id.